docker-compose.yml

```diff
@@ -51,11 +51,11 @@ services:
   mongo-merkle-tree:
     image: mongo
     environment:
-      - MONGO_INITDB_ROOT_USERNAME=admin
-      - MONGO_INITDB_ROOT_PASSWORD=admin
+      #- MONGO_INITDB_ROOT_USERNAME=admin
```

Comment:
Looks great! Can we remove these commented lines out? It could confuse people who think that we have an optional authentication implemented or something like that.

Reply:
Good point - done!